cooltools/api/coverage.py

```diff
@@ -102,24 +125,40 @@ def coverage(
         raise ValueError(
             "Please, specify ignore_diags and/or IC balance this cooler! Cannot access the value used in IC balancing. "
         )
-
+    if clr_weight_name is not None:
+        try:
+            _ = is_cooler_balanced(clr, clr_weight_name, raise_errors=True)
+        except Exception as e:
+            raise ValueError(
+                f"provided cooler is not balanced or {clr_weight_name} is missing"
+            ) from e
+            
```

Comment:
maybe here or in a separate `if clr_weight_name` clause we could add the `bias` and `na_mask` initialization, like so:
```python
if clr_weight_name is not None:
    bias = clr.bins()[clr_weight_name][:]
    bias_na_mask = np.isnan(bias)  # remember masked bins
    bias = np.nan_to_num(bias)  # replace NaNs with 0 in bias to enable summation
```